src/strands/experimental/bidirectional_streaming/agent/agent.py

```diff
@@ -431,6 +431,75 @@ async def end(self) -> None:
             await stop_bidirectional_connection(self._session)
             self._session = None
 
+    async def __call__(
```

Comment:
Contrary to @Unshure I would have preferred `run` - if only because this has different behavior than the single-agent's `__call__` - e.g. this is not starting a "loop" of the agent, it's actually kick starting the long running process.

Not a blocker, but I think `run` might be more clear for that reason

Reply:
I have been thinking about this, and I agree. 

Long term, I can see bidi connections with any-to-any models becoming standards, and I think we should reserve `__call__` for the same use case as we have in normal agents, i.e. `__call__` should execute the agent and return a response.

I will keep `run`  method